Cloudflare/zerotrust_dns/_meta/manifest.yml

```
@@ -0,0 +1,11 @@
+uuid: 7b1317ec-3f87-4b53-9b6d-3f79045f28fa
+name: ZeroTrust DNS
```

Use the name of the dataset from Cloudflare
```suggestion
name: Cloudflare Gateway DNS
```